UE/Tests/Application/Ports/BtsPortTestSuite.cpp

```diff
@@ -102,4 +102,17 @@ TEST_F(BtsPortTestSuite, shallSendAttachRequest)
     ASSERT_NO_THROW(reader.checkEndOfMessage());
 }
 
+TEST_F(BtsPortTestSuite, shallSendCallAccepted)
+{
+    constexpr common::PhoneNumber to_phone_number{113};
+    common::BinaryMessage msg;
+    EXPECT_CALL(transportMock, sendMessage(_)).WillOnce([&msg](auto param) { msg = std::move(param); return true; });
+    objectUnderTest.sendCallAccepted(to_phone_number);
+    common::IncomingMessage reader(msg);
+    ASSERT_NO_THROW(EXPECT_EQ(common::MessageId::CallAccepted, reader.readMessageId()));
+    ASSERT_NO_THROW(EXPECT_EQ(PHONE_NUMBER, reader.readPhoneNumber()));
+    ASSERT_NO_THROW(EXPECT_EQ(to_phone_number, reader.readPhoneNumber()));
```

Comment:
Zastanawiam się czy potrzebny jest ASSERT_NO_THROW. Nie mówię że jest to źle.

Reply:
I'm not sure either, but I don't think that it will hurt anyhow.